luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialPark.java

```diff
@@ -409,8 +413,35 @@ public Builder(LuthierResourceDictionaries resourceDictionaries, GranularityDict
             return logicalTableFactoryMap;
         }
 
-        public GranularityDictionary getGranularityDictionary() {
```

Comment:
Why are we getting rid of this?

Reply:
yes, why?

Reply:
I was under the impression that the only communication regarding Granularity between the LIP and various Factories is one-directional. i.e. LIP gives information to Factories and Factories never have the privilege to modify the LIP's GranularityDictionary. 
This change highlights the fact that this means that the GranularityParser can hide away the implementation of GranularityDictionary inside LIP. So far, none of the Factories are calling GranularityDictionary, as that information is exposed by the GranularityParser itself. 
If user need to modify the GranularityParser, they should do it by subclassing LuthierBinderFactory thus have full control over what the GranularityDictionary is. 

Please feel free to point out the flaw in this thinking.

Reply:
during the builder building erring on the side of mutability is your friend.

Reply:
My bad, this is not in the LIP, this is the LIP.Builder. I have put it back.